tests/integrationv2/test_signature_algorithms.py

```diff
@@ -2,12 +2,11 @@
 import pytest
 
 from configuration import available_ports, ALL_TEST_CIPHERS, ALL_TEST_CERTS
-from common import ProviderOptions, Protocols, Certificates, Signatures, data_bytes
-from fixtures import managed_process # lgtm [py/unused-import]
+from common import ProviderOptions, Protocols, Certificates, Signatures, data_bytes, Ciphers
+from fixtures import managed_process  # lgtm [py/unused-import]
 from providers import Provider, S2N, OpenSSL, GnuTLS
 from utils import invalid_test_parameters, get_parameter_name, get_expected_s2n_version, to_bytes
 
-
 certs = [
```

Comment:
Nit: are these whitespace changes intentional?

Reply:
Autopep8 will now fail if it doesn't like the whitespace here. See: https://github.com/aws/s2n-tls/pull/3736